mr/src/main/java/org/apache/mahout/cf/taste/impl/similarity/precompute/MultithreadedBatchItemSimilarities.java

```diff
@@ -26,7 +26,7 @@
 import java.util.concurrent.TimeUnit;
 import java.util.concurrent.atomic.AtomicInteger;
 
-import com.google.common.collect.Lists;
+import java.util.ArrayList;
 import com.google.common.io.Closeables;
```

Comment:
I am seeing github issues too, consistently been getting RPC timeouts for
the past 30 mins.

On Sun, Apr 5, 2015 at 3:20 AM, Anand Avati <notifications@github.com>
wrote:

> In
> mr/src/main/java/org/apache/mahout/cf/taste/impl/similarity/precompute/MultithreadedBatchItemSimilarities.java
> <https://github.com/apache/mahout/pull/106#discussion_r27775438>:
>
> > @@ -26,7 +26,7 @@
> >  import java.util.concurrent.TimeUnit;
> >  import java.util.concurrent.atomic.AtomicInteger;
> >
> > -import com.google.common.collect.Lists;
> > +import java.util.ArrayList;
> >  import com.google.common.io.Closeables;
>
> For some reason there is a delay in GitHub reflecting branch changes in PR.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/apache/mahout/pull/106/files#r27775438>.
>